webapp/src/Controller/Jury/ProblemController.php

```diff
@@ -477,8 +478,18 @@ public function viewTextAction(int $probId): StreamedResponse
     }
 
     #[Route(path: '/{probId<\d+>}/testcases', name: 'jury_problem_testcases')]
-    public function testcasesAction(Request $request, int $probId): Response
-    {
+    public function testcasesAction(
+        Request $request,
+        int $probId,
+        #[MapQueryParameter(name: 'testcase')]
+        ?int $testcaseFromRequest = null,
+    ): Response {
+        if ($testcaseFromRequest) {
+            $tc = $this->em->getRepository(Testcase::class)->find($testcaseFromRequest);
+            if ($tc) {
+                return $this->redirectToRoute('jury_problem_testcases', ['probId' => $tc->getProblem()->getProbId()]);
```

Comment:
I have the impression that this behavior can be very confusing. You are requesting one problem with a testcase from a different problem and are redirected?

Perhaps it becomes less confusing if we just add a separate route for it (in case it is useful)?

Reply:
So currently it is undocumented behaviour, and I think there are moments where you might want to know what testcase belongs with which problem (there is on the judgehost a system problem and you want to findout what kind of behaviour that problem has (loads of IO, loads of memory etc.)

I'm fine with just removing the commit, or just making a new route.